make_dataset.py

```diff
@@ -483,25 +500,60 @@ def parse_args(args_input=None):
             warnings.warn("Unable to degrade chosen excerpt from "
                           f"{comp.csv_path} and no clean excerpts requested."
                           " Skipping.", UserWarning)
-
+    
     meta_file.close()
+    
+    fh_dict = {
+        split: open(
+            os.path.join(ARGS.output_dir, f'{split}_cmd_corpus.csv'
+        ), 'w') for split in ['train', 'valid', 'test']
+    }
+    line_counts = {
+        split: 0 for split in ['train', 'valid', 'test']
+    }
+    meta_df = pd.read_csv(os.path.join('acme', 'metadata.csv'))
+    for idx, row in tqdm(meta_df.iterrows(), total=meta_df.shape[0],
+                         desc='Creating command corpus'):
+        alt_df = pd.read_csv(os.path.join('acme', row.altered_csv_path),
+                             header=None,
+                             names=['onset', 'track', 'pitch', 'dur'])
+        alt_cmd_str = df_to_command_str(alt_df)
+        clean_df = pd.read_csv(os.path.join('acme', row.clean_csv_path),
+                               header=None,
+                               names=['onset', 'track', 'pitch', 'dur'])
```

Comment:
Does data_structures.read_note_csv not work for this? If not, we need a function that does, I think.

Reply:
not sure I completely follow

Reply:
There's already a function data_structures.read_note_csv that I thought we had written to do this.